### PR TITLE
update srcset

### PR DESCRIPTION
**Pull request by Arhell**

### CODE REVIEW

**Comment by rob006** [comment]
`scrset` should have all versions of images, including 1x. See:

https://github.com/yiisoft/yii2/blob/87a9beaededa9ef785a2003d56d808024481128a/framework/helpers/BaseHtml.php#L453-L459

https://github.com/yiisoft/yii2/blob/4de66f8205fe74d643b010a922a093ce390a5bb2/tests/framework/helpers/HtmlTest.php#L261-L284

File: views/site/partials/tour/2.php [hunk 1]
```diff
@@ -3,7 +3,7 @@
         <div class="col-md-7">
             <img src="<?= Yii::getAlias('@web/image/tour/2.png') ?>"
                  alt="Welcome Page"
-                 srcset="<?= Yii::getAlias('@web/image/tour/2@2x.png') ?>"
+                 srcset="<?= Yii::getAlias('@web/image/tour/2.png 1x, @web/image/tour/2@2x.png 2x') ?>"
```
Comment (rob006) [inline views/site/partials/tour/2.php:6]: It won't work either - `getAlias()` will resolve only first URL.

Comment (samdark) [reply views/site/partials/tour/2.php:6]: I'll fix it.

Comment (samdark) [reply views/site/partials/tour/2.php:6]: https://github.com/yiisoft-contrib/yiiframework.com/commit/456e5371a1744bdcd21881333aa154937f15964e

**Pull request merged**